pkg/storer/internal/reserve/items.go

```diff
@@ -104,11 +102,11 @@ func (b *BatchRadiusItem) Unmarshal(buf []byte) error {
 // ChunkBinItem allows for iterating on ranges of bin and binIDs for chunks.
 // BinIDs come in handy when syncing the reserve contents with other peers.
 type ChunkBinItem struct {
-	Bin       uint8
-	BinID     uint64
-	Address   swarm.Address
-	BatchID   []byte
-	ChunkType swarm.ChunkType
+	Bin     uint8
+	BinID   uint64
+	Address swarm.Address
+	BatchID []byte
+	Type    swarm.ChunkType
```

Comment:
Is this rename really necessary for the new transaction? It adds additional changes that may not be needed.

Reply:
sure, reverted